app/containers/Forms/FormCreateUser.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {Row, Col, Form, Button} from 'react-bootstrap';
+import {createFragmentContainer} from 'react-relay';
+import createUserMutation from '../../mutations/user/createUserMutation';
+
+export const FormCreateUser = props => {
+  const user = {
+    firstName: '',
+    lastName: '',
+    emailAddress: '',
+    occupation: '',
+    phoneNumber: ''
+  };
+
+  const createUserFromRef = React.createRef();
```

Comment:
Why do we need a ref?

Reply:
To clear out the form after submission. But maybe i can hook up using id instead of ref.

Reply:
Why are you clearing out the form after submission? Shouldn't we redirect the user to the user dashboard after they do the onboarding?

Reply:
If we redirect, where are we handling validations and showing the errors? The flow is not finalized so for now I had it just to clear the form. We might need to add 'Next' button to the flow.